src/dns_message.c

```diff
@@ -54,6 +54,7 @@
 #endif
 
 
+#include "config.h"
```

Comment:
Because some lines below you have:
#if HAVE_LIBGEOIP
 #include "country_index.h"

But HAVE_LIBGEOIP is defined in config.h and not yet included.